src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/MapReferenceScale/readme.md

```diff
@@ -12,23 +12,23 @@ Map annotations are typically only relevant at certain scales. For instance, ann
 
 ## How to use the sample
 
-* Use the selection box to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000).
-* Use the list to choose which layers participate in symbol scaling.
+* Use the control at the top to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000).
+* Use the menu checkboxes in the layer menu to set which feature layers should honor the reference scale.
```

Comment:
I think the how-to-use section isn't supposed to use lists; may need to be changed in the common design.